csv_to_json_large_convos_to_single_utterance.py

```diff
@@ -0,0 +1,259 @@
+"""
+python csv_to_json_combine_large_convos.py
+
+"""
+# *********************************************************************************************************************
+import click
```

Comment:
Split the libraries into standard, 3rd party and custom imports. For example, see https://github.com/zia-ai/academy/blob/master/ada_to_csv.py